.github/workflows/windows.yml

```diff
@@ -51,11 +52,40 @@ jobs:
           done
           echo "Extracting $files"
           7z x LLVM-13.0.1-win64.exe $files
+      # MinGW is used only for linking the compiled executables.
```

Comment:
Mentioned in a couple other places already. More repetition --> more likely to get outdated
```suggestion
```